src/test/java/seedu/address/logic/commands/GetMonthlyFeeCommandTest.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.testutil.TypicalStudents.ALICE;
+import static seedu.address.testutil.TypicalStudents.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.fee.Month;
+import seedu.address.model.fee.Year;
+
+public class GetMonthlyFeeCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    // Alice has 4 session and has only 2 session that is on 1 Jan 2021
+    // The other 2 session, one has a different month, one has a different year
+    @Test
+    public void execute_correctMonthlyFee() throws CommandException {
```

Comment:
maybe `execute_correctMonthlyFee_success()` would be better.

Reply:
Haha ok sure! Updated.